profiling/README.md

```diff
@@ -2,18 +2,33 @@
 
 The performance tests are designed to hit the APIs using the [Vegeta utility](https://github.com/tsenart/vegeta)
 
-## Running the perf test
-The [sync_to_user](./sync_to_user) file contains the GET APIs that would be hit as part of the performance test. The headers and body parameters can also be configured in here.
+## Running the performance test
+There's a template file called [sync_to_user.example](./sync_to_user.example) which defines how you can specify the APIs that need to be hit as a part of the performance test. The credentials are all crossed-out.
+
+Copy it over to a new file called [sync_to_user](./sync_to_user) via,
+
+```
+make gen-requests for=sync_to_user
+```
+
+Configure the headers and body parameters appropriately.
+
+The performance test itself can be run as follows:
 
-The perf test itself can be run as follows:
 ```
 make plot request=sync_to_user duration=20 rate=1
 ```
-This will run the perf test at the breakneck speed of 1 request per second for 20 seconds and open up (this will only work on OS X) a latency report page for you.
+
+This will run the test at the breakneck speed of 1 request per second for 20 seconds and open up (this will only work on OS X) a latency report page for you.
```

Comment:
Your wit is undeniably entertaining, but perhaps we could omit it from the readme? 😅 (re: breakneck speed)